pkg/cli/node.go

```diff
@@ -595,7 +595,7 @@ func runRecommissionNode(cmd *cobra.Command, args []string) error {
 }
 
 var drainNodeCmd = &cobra.Command{
-	Use:   "drain",
+	Use:   "drain [ <node id> ]",
```

Comment:
Should this take a list of node IDs instead? For symmetry with `cockroach node decommission`. Internally (probably at the CLI level) we could step through them one-by-one.

Reply:
UX question: In the interest of symmetry, what do you think of introducing a --self flag? We'd keep the "empty list drains self behavior" for now, for compatibility reasons, but perhaps deprecate it in favor of the more explicit --self. This is motivated only by way of symmetry with decommission, where the host being connected to in order to drain is not implicitly assumed to be the drain target.